zap/src/main/java/org/zaproxy/zap/extension/api/ApiDynamicActionImplementor.java

```diff
@@ -41,6 +42,21 @@ public ApiDynamicActionImplementor(
         super(name, mandatoryParamNames, optionalParamNames);
     }
 
+    /**
+     * Instantiates a new api dynamic action implementor.
+     *
+     * @param name the name
+     * @param mandatoryParamNames the mandatory param names, or <code>null</code> if there are no
+     *     mandatory parameters
+     * @param optionalParamNames the optional param names, or <code>null</code> if there are no
+     *     optional parameters
```

Comment:
This says null but then in the next classes use List.of

Reply:
One thing does not prevent the other, it also disambiguates.

Reply:
Ok